module1.js

```diff
@@ -54,6 +67,11 @@ function question3(number) {
 
 function question4(array) {
   // TODO:
+  var j = 0;
+  for (var i = 0; i < array.length; i + 2) {
+    array[j] = array[i];
```

Comment:
Not sure how this was meant to work but this question was tough - so maybe you can try again?